src/scrabble.js

```diff
@@ -1,5 +1,191 @@
-function scrabble() {
+function normalScore(aWord) {
+  if (aWord === '' || aWord === ' \t\n' || aWord === null) {
+    return 0
+  } else {
+    const newWord = aWord
+    console.log('new word is', newWord)
+    let sum = 0
+    for (i = 0; i < newWord.length; i++) {
+      if (
+        newWord[i].includes('A') ||
+        newWord[i].includes('E') ||
+        newWord[i].includes('I') ||
+        newWord[i].includes('O') ||
+        newWord[i].includes('U') ||
+        newWord[i].includes('L') ||
+        newWord[i].includes('N') ||
+        newWord[i].includes('R') ||
+        newWord[i].includes('S') ||
+        newWord[i].includes('T') ||
+        newWord[i].includes('a') ||
+        newWord[i].includes('e') ||
+        newWord[i].includes('i') ||
+        newWord[i].includes('o') ||
+        newWord[i].includes('u') ||
+        newWord[i].includes('l') ||
+        newWord[i].includes('n') ||
+        newWord[i].includes('r') ||
+        newWord[i].includes('s') ||
+        newWord[i].includes('t')
+      ) {
+        sum += 1
+      }
+      if (
+        newWord[i].includes('D') ||
+        newWord[i].includes('G') ||
+        newWord[i].includes('d') ||
+        newWord[i].includes('g')
+      ) {
+        sum += 2
+      }
+      if (
+        newWord[i].includes('B') ||
+        newWord[i].includes('C') ||
+        newWord[i].includes('M') ||
+        newWord[i].includes('P') ||
+        newWord[i].includes('b') ||
+        newWord[i].includes('c') ||
+        newWord[i].includes('m') ||
+        newWord[i].includes('p')
+      ) {
+        sum += 3
+      }
+      if (
+        newWord[i].includes('F') ||
+        newWord[i].includes('H') ||
+        newWord[i].includes('V') ||
+        newWord[i].includes('W') ||
+        newWord[i].includes('Y') ||
+        newWord[i].includes('f') ||
+        newWord[i].includes('h') ||
+        newWord[i].includes('v') ||
+        newWord[i].includes('w') ||
+        newWord[i].includes('y')
+      ) {
+        sum += 4
+      }
+      if (newWord[i].includes('K') || newWord[i].includes('k')) {
+        sum += 5
+      }
+      if (
+        newWord[i].includes('J') ||
+        newWord[i].includes('X') ||
+        newWord[i].includes('j') ||
+        newWord[i].includes('x')
+      ) {
+        sum += 8
+      }
+      if (
+        newWord[i].includes('Q') ||
+        newWord[i].includes('Z') ||
+        newWord[i].includes('q') ||
+        newWord[i].includes('z')
+      ) {
+        sum += 10
+      }
+    }
+    return sum
+  }
+}
+
+function doubleLetter(aWord) {
+  const regex1 = /{[A-Z|a-z]}/g
```

Comment:
Really nice idea to use regex for your double and triple letter checks. Really clever and much more efficient than my method of using if loops. Great idea.